go-chaos/cmd/verify.go

```diff
@@ -33,6 +36,8 @@ func init() {
 	verifySteadyStateCmd.Flags().StringVar(&processModelPath, "processModelPath", "", "Specify the path to a BPMN process model, which should be deployed and an instance should be created of.")
 	verifySteadyStateCmd.Flags().StringVar(&variables, "variables", "", "Specify the variables for the process instance. Expect json string.")
 	verifySteadyStateCmd.Flags().BoolVar(&awaitResult, "awaitResult", false, "Specify whether the completion of the created process instance should be awaited.")
+	verifySteadyStateCmd.Flags().IntVar(&version, "version", -1, "Specify the version for which the instance should be created.")
```

Comment:
:question: Is there a way to mark these flags as optional and depending on each other? I fear that as we are adding more options, it's becoming less clear what flags need to be set and what their dependencies are. For example, it's not clear that when `version` is set, `bpmnProcessId` is required and `processModelPath` has no effect.

Reply:
I will follow up on this https://github.com/zeebe-io/zeebe-chaos/issues/249